src/render-instruction.ts

```diff
@@ -116,13 +117,38 @@ ${typeMapperImports.join('\n')}`.trim()
   // Accounts
   // -----------------
   private processIxAccounts(): ProcessedAccountKey[] {
-    return this.ix.accounts.map((acc) => {
-      const knownPubkey = resolveKnownPubkey(acc.name)
-      const optional = acc.optional ?? false
-      return knownPubkey == null
-        ? { ...acc, optional }
-        : { ...acc, knownPubkey, optional }
+    let processedAccountsKey: ProcessedAccountKey[] = []
+    this.ix.accounts.map((acc: IdlInstructionAccount | IdlAccountsCollection) => {
+      if(this.isAccountsCollection(acc)){
+        acc.accounts.map((ac) => {
```

Comment:
If you're not assigning the result of `map` then you should use `forEach` instead.

But I prefer `for (const acc of acc.accounts)`.